tests/testthat/helper.R

```diff
@@ -0,0 +1,19 @@
+#' utility function to make sure connection is closed after usage
+with_dbc_connection <- function(connection, code) {
+  on.exit({
+    DatabaseConnector::disconnect(connection)
+  })
+  eval(substitute(code), envir = connection, enclos = parent.frame())
+}
+
+#' Only works with postgres > 9.4
+.pgTableExists <- function(connection, schema, tableName) {
```

Comment:
Couldn't we use [`DatabaseConnector::getTableNames()`](http://ohdsi.github.io/DatabaseConnector/reference/getTableNames.html) here to check if the table exists?

Reply:
I think this is actually some legacy hack where there was a problem calling `getTableNames` between different database connections (the tables are created with a different connection, probably to increase test coverage), I'll try to remove this.

Reply:
Update: use of getTableNames failed in a previous commit, I'm not sure why but it's possibly related to the information schema or some cache being refreshed slower than these tests require. For this reason I reverted the change.